Lets Do This/Info.plist

```diff
@@ -36,6 +38,19 @@
 	</dict>
 	<key>LSRequiresIPhoneOS</key>
 	<true/>
+	<key>NSAppTransportSecurity</key>
+	<dict>
+		<key>NSAllowsArbitraryLoads</key>
```

Comment:
Another todo: Restrict the domains to only the LTD News WP domains? Or do we need this here in order to render any full article URL in the `LDTNewsArticleViewController` ?